src/main/java/org/jenkinsci/plugins/emotional_jenkins/EmotionalJenkinsAction.java

```diff
@@ -7,8 +7,6 @@
 
     private Result result;
 
-    public EmotionalJenkinsAction() {}
```

Comment:
Uh? This constructor seems to be used for "no result" purposes. It's deletion breaks the binary compatibility